pkg/externalresource/control.go

```diff
@@ -99,9 +99,13 @@ func translate(name string, externalResource *ExternalResource) *k8s.External {
 			APIVersion: fmt.Sprintf("%s/%s", k8s.GroupName, k8s.GroupVersion),
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name: format.ResourceK8sMetaString(name),
+			Name:      format.ResourceK8sMetaString(name),
+			Namespace: ns,
```

Comment:
We were not specifying the namespace but now that we have it, it doesn't hurt to include it